docs/src/man/getting_started.md

```diff
@@ -388,6 +388,17 @@ julia> df[!, r"x"]
 │ 1   │ 1     │ 2     │
 ```
 
+In addition, [InvertedIndices](https://github.com/mbauman/InvertedIndices.jl) can be used to select all columns excluding a specified subset:
```

Comment:
```suggestion
A `Not` selector (from the [InvertedIndices](https://github.com/mbauman/InvertedIndices.jl) package) can be used to select all columns excluding a specific subset:
```
It would also be nice to show an example with a vector of columns (as it's not obvious that you can do that).